morf-core/src/main/java/org/alfasoftware/morf/jdbc/SqlScriptExecutorProvider.java

```diff
@@ -31,19 +32,23 @@ public class SqlScriptExecutorProvider implements Provider<SqlScriptExecutor> {
 
   private final DataSource dataSource;
   private final Provider<SqlDialect> sqlDialect;
+  private ConnectionResources connectionResources;
 
   /**
    * Constructor for Guice.
    *
    * @param dataSource The {@link DataSource} to instantiate the
    *          {@link SqlScriptExecutorProvider} for
    * @param sqlDialect The dialect to use
+   * @param sqlDialect The {@link ConnectionResources} to instantiate the
+   *                   {@link SqlScriptExecutorProvider} for
    */
   @Inject
-  public SqlScriptExecutorProvider(final DataSource dataSource, Provider<SqlDialect> sqlDialect) {
+  public SqlScriptExecutorProvider(final DataSource dataSource, Provider<SqlDialect> sqlDialect, ConnectionResources connectionResources) {
     super();
     this.dataSource = dataSource;
     this.sqlDialect = sqlDialect;
+    this.connectionResources = connectionResources;
   }

```

Comment:
Important: This is a backwards-incompatible change.
Because this is a public constructor, it is possible some code out there is calling this directly.
Don't let the `@Inject` annotation fool you, it can still be used as a regular constructor.